packages/contracts/deploy/00_managing-dao.ts

```diff
@@ -17,7 +17,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
     proxy: {
       owner: deployer,
-      proxyContract: 'UUPSProxy',
+      proxyContract: 'PluginERC1967Proxy',
```

Comment:
is it possible this change is not included in this PR?
it is already done in another PR pending review at :
https://github.com/aragon/core/pull/71

same for changes on : 05_ens.ts, 10_dao-registry.ts, 05_aragon-plugin-registry.ts

Reply:
DONE